drizzlepac/alignimages.py

```diff
@@ -888,27 +860,27 @@ def determine_fit_quality(imglist, filtered_table, catalogs_remaining, print_fit
         fit_rms_val = item.meta['fit_info']['FIT_RMS']
         max_rms_val = item.meta['fit_info']['TOTAL_RMS']
         num_xmatches = item.meta['fit_info']['nmatches']
-        fit_status_dict[dict_key]['max_rms'] = max_rms_val
-        fit_status_dict[dict_key]['num_matches'] = num_xmatches
+        fit_status_dict[dictKey]['max_rms'] = max_rms_val
+        fit_status_dict[dictKey]['num_matches'] = num_xmatches
 
         if num_xmatches < MIN_CROSS_MATCHES:
             if catalogs_remaining:
                 log.warning(
-                    "Not enough cross matches found between astrometric "
+                    "Not enough cross matches found between astrometric"
                     "catalog and sources found in {}".format(image_name))
                 continue
 
         # Execute checks
-        nmatches_check = False
+        nmatchesCheck = False
```

Comment:
Variable should not be camelcase. Rename 'nmatchesCheck' to 'nmatches_check and find/replace the remaining instances.

Reply:
Resolved in 51e3cea